server/mongoose.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose');
+
+const MONGODB_URI = 'mongodb://localhost:27017/pesto';
```

Comment:
This should be a variable that is stored in a config file, and in that config file, it should use environment variables